misc/identity-keys/test/identity-keys.test.ts

```diff
@@ -1,7 +1,87 @@
 import "mocha";
+import "isomorphic-unfetch";
```

Comment:
Could probably just use axios (which is already a dep) for the `fetch` call needed in here but will leave it up to you 👍

Reply:
yep, good idea